bin/stashcp

```diff
@@ -141,6 +143,19 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
         es_send(payload)
 
 
+def parse_job_ad():
+    temp_list = {}
+    try:
+        with open('.job.ad') as job_file:
```

Comment:
Reading in from the environment was causing the build to fail beforehand, even though I had the exact name of the condor environment variable